# Routine CEO/CFO appointments → covered-call incremental value

Separate hypothesis-test copy using the starter library and the latest main-branch research context.
Run all cells from the project folder with `MASSIVE_API_KEY` in `.env`. Code is in
`covered_call_hypothesis.py`; the unchanged prior event-classification implementation is in
`leadership_hypothesis.py`. Both files and the original starter must be alongside this notebook.

**Primary prediction:** positive event-minus-ordinary mean **incremental net value over stock alone**
at 21 sessions. Test CEO appointments and CFO appointments separately; explicit resignations are secondary.

Buy 100 shares and sell one starter 5% OTM call, 3–6 months to expiry. At any close before expiry,
incremental gross value is `(entry call premium − exit call value) / entry stock price`.
The stock holding and its dividends cancel against the identical stock-alone benchmark. Absolute covered
returns shown here exclude dividends; they are not the primary outcome. At 21 sessions the call's
buyback value includes remaining time value; it is not merely intrinsic upside surrendered.

## Frozen safeguards

* Observed Massive stock bars replace the prior parity proxy. Exclude splits during the holding window;
  require an entry-day call trade and standard 100-share contracts. Reuse starter expiry and strike
  algorithms. Entry is next trading-session close, conservatively after any after-hours filing.
* Routine classification is exactly the prior strict public-at-entry rule. Unknown is not routine.
  Same-filing evidence, deduplication, and exact taxonomy snapshots are saved. The teammate's frozen
  JEV low score is a **proxy sensitivity**, not proof of routine status or a new primary definition.
* Ordinary matching preserves the previous company/year/quarter/weekday, 63-session radius,
  30-calendar-day exclusion around any tagged disclosure, DTE tolerance of seven days, and nearest
  three usable controls. A predefined cleaner-entry/leadership-only exclusion sensitivity diagnoses
  whether the broad exclusion itself destroys coverage; it never replaces the primary comparison.
* Count usable primary events before sensitivity or inference. Report all original horizons and expiry
  close, costs 0/2.5/5/10% premium per side plus $0.65 per contract per side, strikes 3/5/10% OTM,
  buckets 1m/2m/3–6m, entry delay 0/1, exit staleness 0/3, routine/all/JEV-low categories, and both
  ordinary definitions. Small samples do not establish no effect.
* Primary 97.5% cluster intervals cover the two appointment tests and resample connected dependence
  components linking repeated companies, reused controls, and overlapping holding intervals across
  companies. Below five components no reliable primary CI is issued. Company-only bootstrap CIs are
  diagnostics and do not protect against cross-company synchronized overlap.
* Compare premiums, buyback value, intrinsic upside surrendered, terminal/path upside and downside,
  absolute moves, ±5% tails, extreme outcomes, and earnings proximity. Decomposition is descriptive;
  neither a positive premium credit nor lower average upside establishes causal mispricing.
* One contract requires 100 shares. Capacity diagnostic is 1% of the smaller entry/exit daily option
  volume, rounded down. This is a volume participation assumption, not displayed liquidity. No quote
  spreads, early assignment or dividend-driven exercise are modeled; mark-to-market results remain
  theoretical. Daily marks can be stale by the specified rule.
* The historical windows were viewed in prior local/team experiments. Report them as historical
  validation, disclose earlier tests, and reserve genuine replication for judges' untouched dates.
  `study.run_study(start, end, label='sealed', output='covered_call_sealed')` accepts those dates.
  The sealed prediction, frozen before this test: **fragile/inconclusive**, with small samples, costs,
  asymmetric upside tails and sparse ordinary controls; positive replication is not presumed.

The main-branch harness, JEV scorer/cache, historical ledger and original notebook are not changed by
this test. No protective-put, collar or cash-secured-put hypothesis is run.


In [1]:
import json
from pathlib import Path
import covered_call_hypothesis as study
study.initialize()


1655 sessions in the calendar; last completed session 2026-10-02. Sessions per year: 2022: 251, 2023: 250, 2024: 252, 2025: 250, 2026: 251


## Freeze choices before pricing

In [2]:
out = Path('covered_call_results')
out.mkdir(exist_ok=True)
protocol = study.freeze(out)
print(json.dumps(protocol, indent=2))


{
  "bootstrap_replicates": 5000,
  "buckets": [
    "1m",
    "2m",
    "3-6m"
  ],
  "candidate_cap": 120,
  "capacity": "one covered contract =100 shares; diagnostic maximum contracts=floor(1% min(entry volume,exit volume))",
  "class_grid": [
    "routine",
    "all",
    "jev_low_proxy"
  ],
  "classification": "unchanged strict entry-evidence classifier from prior leadership test; unknown not routine",
  "commission_per_contract_per_side": 0.65,
  "confidence": 0.975,
  "construction": "starter expiry/strike algorithms on entry-day chain and observed entry stock price; buy 100 shares, sell one 5% OTM standard call",
  "control_rules": [
    "all_disclosures_30d",
    "entry_clean_leadership_30d"
  ],
  "controls_per_event": 3,
  "costs": [
    0,
    0.025,
    0.05,
    0.1
  ],
  "earnings": "record tagged earnings within seven calendar days at entry as a partial confound indicator",
  "entry": "first session close strictly after filing_date; next-close delay sensitivity",
  "e

## Execute counts, in-sample and historical out-of-sample

In [3]:
summary, board, status = study.main()
print(json.dumps(status, indent=2))


Exact verified tags: {'ceo_appointment': 'ceo_appointment', 'ceo_resignation': 'ceo_departure', 'cfo_appointment': 'cfo_appointment', 'cfo_resignation': 'cfo_departure'}
in_sample candidate counts BEFORE pricing: [{'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'raw_disclosures', 'n': 968}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'outside_universe_disclosures', 'n': 951}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'duplicate_shareclass_or_excerpt_rows', 'n': 0}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'repeated_or_retrospective_appointment', 'n': 2}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'classification_routine', 'n': 10}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'classification_nonroutine', 'n': 1}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'classification_unknown', 'n': 4}, {'window': 'in_sample', 'group': 'ceo_appointment', 'stage': 'candidate_events

## All fixed horizons, primary specification

In [4]:
display(board)


,window,group,horizon,n_events,difference,ci_lo,ci_hi,dependence_clusters,bucket,delay,otm,stale,cost,control_rule,class_filter,n_companies,company_only_ci_lo,company_only_ci_hi,incremental_net,ordinary_incremental_net,covered_net,entry_premium,ordinary_entry_premium,exit_premium,ordinary_exit_premium,stock_return,ordinary_stock_return,absolute_move,ordinary_absolute_move,upside,ordinary_upside,downside,ordinary_downside,upside_tail,ordinary_upside_tail,downside_tail,ordinary_downside_tail,max_upside,max_downside,intrinsic_upside_surrendered,ordinary_intrinsic_upside_surrendered,call_buyback_value,ordinary_call_buyback_value,entry_volume,exit_volume,capacity_contracts,moneyness,event_earnings_near,ordinary_earnings_near,entry_premium_gap,buyback_value_gap,upside_gap,stock_return_q0.05,stock_return_q0.5,stock_return_q0.95,largest_upside,worst_downside
0,in_sample,ceo_appointment,1,3,0.005783,NaN,NaN,3,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,-0.001402,-0.007186,-0.001941,0.041716,0.045681,0.038998,0.048106,-0.000539,0.009094,0.001792,0.009094,0.000626,0.009094,-0.001165,0.000000,0.000000,0.000000,0.000000,0.000000,0.000626,-0.001165,0.000000,0.000000,0.038998,0.048106,39.333333,72.000000,0.000000,0.060403,0.0,0.0,-0.003965,-0.009108,-0.008468,-0.003067,0.000789,0.001060,0.001090,-0.003495
1,in_sample,ceo_appointment,2,3,0.002603,NaN,NaN,3,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,-0.002551,-0.005155,-0.000201,0.041716,0.045681,0.040093,0.046172,0.002351,0.001423,0.012382,0.012021,0.007366,0.006722,-0.005016,-0.005299,0.000000,0.000000,0.000000,0.000000,0.007730,-0.005016,0.000000,0.000000,0.040093,0.046172,39.333333,64.000000,0.333333,0.060403,0.0,0.0,-0.003965,-0.006079,0.000644,-0.010646,-0.003621,0.019527,0.022099,-0.011427
2,in_sample,ceo_appointment,3,3,0.010103,NaN,NaN,3,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,0.000280,-0.009823,-0.011259,0.041716,0.045681,0.037396,0.050618,-0.011539,0.025729,0.021431,0.034983,0.004946,0.030356,-0.016485,-0.004627,0.000000,0.166667,0.000000,0.000000,0.007730,-0.016485,0.000000,0.015852,0.037396,0.050618,39.333333,67.333333,0.333333,0.060403,0.0,0.0,-0.003965,-0.013222,-0.025410,-0.028409,-0.020127,0.011342,0.014838,-0.029329
3,in_sample,ceo_appointment,5,3,0.011460,NaN,NaN,2,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,-0.003898,-0.015358,-0.000131,0.041716,0.045681,0.041375,0.055889,0.003767,0.023880,0.032227,0.027068,0.017997,0.025474,-0.014230,-0.001594,0.333333,0.166667,0.000000,0.000000,0.017997,-0.020939,0.000000,0.005019,0.041375,0.055889,39.333333,11.000000,0.000000,0.060403,0.0,0.0,-0.003965,-0.014514,-0.007477,-0.038310,0.001110,0.047704,0.052881,-0.042690
4,in_sample,ceo_appointment,10,2,-0.033331,NaN,NaN,2,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,2.0,NaN,NaN,-0.029761,0.003570,0.029137,0.039952,0.028985,0.064381,0.022737,0.058898,-0.024185,0.078908,0.025698,0.068903,0.000757,-0.010005,-0.024941,0.500000,0.000000,0.000000,0.250000,0.069458,-0.010063,0.036148,0.000000,0.064381,0.022737,57.500000,28.000000,0.000000,0.058312,0.0,0.0,0.010967,0.041645,0.068146,-0.012119,0.058898,0.129915,0.137806,-0.020010
5,in_sample,ceo_appointment,21,3,-0.019215,NaN,NaN,2,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,-0.002488,0.016727,-0.008253,0.041716,0.040003,0.040032,0.020195,-0.005765,-0.041892,0.084481,0.044038,0.039358,0.001073,-0.045123,-0.042965,0.333333,0.000000,0.333333,0.333333,0.048989,-0.046674,0.017522,0.000000,0.040032,0.020195,39.333333,87.666667,0.000000,0.060403,0.0,0.0,0.001714,0.019837,0.038285,-0.086624,-0.044011,0.101866,0.118074,-0.091359
6,in_sample,ceo_appointment,42,3,-0.081710,NaN,NaN,2,3-6m,0.0,0.05,3.0,0.05,all_disclosures_30d,routine,3.0,NaN,NaN,-0.054106,0.027604,-0.010364,0.041716,0.040003,0.089192,0.009837,0.043742,-0.020010,0.146103,0.029205,0.094922,0.004597,-0.051181,-0.024608,0.333333,0.000000,0.666667,0.333333,0.102501,-0.056657,0.073086,0.000000,

## Final assessment

In [5]:
if (out / 'RESULTS.md').exists():
    print((out / 'RESULTS.md').read_text(encoding='utf-8'))
else:
    print('See run_status.json and the saved baseline and sensitivity tables. Final assessment follows execution.')


# Covered-call submission assessment

**Conclusion: inconclusive.**

Hypothesis: routine CEO/CFO appointments increase the net value added by a covered call over stock alone relative to equivalent covered calls on ordinary days. Prediction: positive event-minus-ordinary mean incremental value at 21 sessions. CEOs and CFOs remain separate tests.

The latest `main` was pulled before this experiment (input commit `0cc554f`). A `.gitignore` conflict was resolved in favor of main; prior local work remains recoverable. Actual stock bars were successfully retrieved from Massive. The starter universe, windows, expiry buckets and strike selection rules were retained. The original starter, teammate harness, JEV rules and ledger were not edited by this experiment.

## Primary result: 21 sessions, 3–6 month expiry, 5% OTM

Figures are percentage points of entry stock notional, after 5% premium haircut on entry and exit and $0.65 commission per contract per side. Incremental value is the option ove

# Covered-call submission assessment

**Conclusion: inconclusive.**

Hypothesis: routine CEO/CFO appointments increase the net value added by a covered call over stock alone relative to equivalent covered calls on ordinary days. Prediction: positive event-minus-ordinary mean incremental value at 21 sessions. CEOs and CFOs remain separate tests.

The latest `main` was pulled before this experiment (input commit `0cc554f`). A `.gitignore` conflict was resolved in favor of main; prior local work remains recoverable. Actual stock bars were successfully retrieved from Massive. The starter universe, windows, expiry buckets and strike selection rules were retained. The original starter, teammate harness, JEV rules and ledger were not edited by this experiment.

## Primary result: 21 sessions, 3–6 month expiry, 5% OTM

Figures are percentage points of entry stock notional, after 5% premium haircut on entry and exit and $0.65 commission per contract per side. Incremental value is the option overlay alone: stock holdings and dividends cancel against the identical stock benchmark.

| Window | Event group | Usable events | Event added net value | Ordinary added net value | Difference | Dependence components |
|---|---|---:|---:|---:|---:|---:|
| in_sample | ceo_appointment | 3 | -0.249% | +1.673% | -1.922% | 2 |
| in_sample | cfo_appointment | 3 | -0.820% | +0.414% | -1.234% | 3 |
| in_sample | ceo_resignation | 0 | unavailable | unavailable | unavailable | 0 |
| in_sample | cfo_resignation | 0 | unavailable | unavailable | unavailable | 0 |
| out_of_sample | ceo_appointment | 0 | unavailable | unavailable | unavailable | 0 |
| out_of_sample | cfo_appointment | 1 | -1.655% | -2.344% | +0.689% | 1 |
| out_of_sample | ceo_resignation | 0 | unavailable | unavailable | unavailable | 0 |
| out_of_sample | cfo_resignation | 0 | unavailable | unavailable | unavailable | 0 |

Primary CIs are issued only with at least five independent connected components linking repeated companies and overlapping event/control holding periods. Missing intervals mean insufficient independent information; they are not zero uncertainty. Company-only bootstrap intervals in the sensitivity table are diagnostics and cannot establish the primary claim.

## Premium and upside accounting

The exact incremental gross value is entry premium minus the call value paid to close. At 21 sessions the call retains time value; treating it as intrinsic payoff would overstate covered-call profitability. A positive difference decomposes into higher initial premium and/or a lower subsequent call buyback value, net of costs. Intrinsic upside surrendered and stock upside are separate descriptive measures, not a causal decomposition.

| Window/group | Event / ordinary premium (% stock) | Event / ordinary buyback value (% stock) | Event / ordinary stock return |
|---|---:|---:|---:|
| in_sample / ceo_appointment | +4.172% / +4.000% | +4.003% / +2.020% | -0.577% / -4.189% |
| in_sample / cfo_appointment | +3.932% / +3.286% | +4.329% / +2.570% | +2.696% / -1.468% |
| out_of_sample / cfo_appointment | +4.162% / +4.368% | +5.338% / +6.180% | +3.966% / +7.003% |

## Every fixed horizon

The audited secondary view excludes CAT's board-member resignation by a former CEO. It is not a current CEO resignation. The frozen tag/proximity-filter output is retained unchanged in `baseline_every_horizon.csv`; corrected secondary coverage is in `audited_baseline_every_horizon.csv`. Appointment results and primary decisions are unchanged. The secondary role error was identified from text, not performance.

| Window | Group | Sessions / expiry close | Usable events | Incremental difference | Primary CI |
|---|---|---:|---:|---:|---|
| in_sample | ceo_appointment | 1 | 3 | +0.578% | unavailable |
| in_sample | ceo_appointment | 2 | 3 | +0.260% | unavailable |
| in_sample | ceo_appointment | 3 | 3 | +1.010% | unavailable |
| in_sample | ceo_appointment | 5 | 3 | +1.146% | unavailable |
| in_sample | ceo_appointment | 10 | 2 | -3.333% | unavailable |
| in_sample | ceo_appointment | 21 | 3 | -1.922% | unavailable |
| in_sample | ceo_appointment | 42 | 3 | -8.171% | unavailable |
| in_sample | ceo_appointment | 63 | 3 | -0.998% | unavailable |
| in_sample | ceo_appointment | exp | 2 | -6.214% | unavailable |
| in_sample | cfo_appointment | 1 | 3 | +0.565% | unavailable |
| in_sample | cfo_appointment | 2 | 3 | +0.373% | unavailable |
| in_sample | cfo_appointment | 3 | 3 | +0.242% | unavailable |
| in_sample | cfo_appointment | 5 | 3 | +0.564% | unavailable |
| in_sample | cfo_appointment | 10 | 3 | +1.569% | unavailable |
| in_sample | cfo_appointment | 21 | 3 | -1.234% | unavailable |
| in_sample | cfo_appointment | 42 | 3 | +5.569% | unavailable |
| in_sample | cfo_appointment | 63 | 3 | +5.246% | unavailable |
| in_sample | cfo_appointment | exp | 3 | -6.190% | unavailable |
| in_sample | ceo_resignation | 1 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 2 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 3 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 5 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 10 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 21 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 42 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | 63 | 0 | unavailable | unavailable |
| in_sample | ceo_resignation | exp | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 1 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 2 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 3 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 5 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 10 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 21 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 42 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | 63 | 0 | unavailable | unavailable |
| in_sample | cfo_resignation | exp | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 1 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 2 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 3 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 5 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 10 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 21 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 42 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | 63 | 0 | unavailable | unavailable |
| out_of_sample | ceo_appointment | exp | 0 | unavailable | unavailable |
| out_of_sample | cfo_appointment | 1 | 1 | -0.850% | unavailable |
| out_of_sample | cfo_appointment | 2 | 1 | -1.319% | unavailable |
| out_of_sample | cfo_appointment | 3 | 1 | +0.021% | unavailable |
| out_of_sample | cfo_appointment | 5 | 1 | +0.289% | unavailable |
| out_of_sample | cfo_appointment | 10 | 1 | -0.999% | unavailable |
| out_of_sample | cfo_appointment | 21 | 1 | +0.689% | unavailable |
| out_of_sample | cfo_appointment | 42 | 1 | +10.900% | unavailable |
| out_of_sample | cfo_appointment | 63 | 1 | -14.590% | unavailable |
| out_of_sample | cfo_appointment | exp | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 1 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 2 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 3 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 5 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 10 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 21 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 42 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | 63 | 0 | unavailable | unavailable |
| out_of_sample | ceo_resignation | exp | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 1 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 2 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 3 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 5 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 10 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 21 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 42 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | 63 | 0 | unavailable | unavailable |
| out_of_sample | cfo_resignation | exp | 0 | unavailable | unavailable |

## Exclusions

Distinct primary event exclusions below are counted once per event. Raw/universe, duplicate and explicit-resignation exclusions are separately saved in each inventory table. Full parameter-row exclusions contain control failures and repeated sensitivity rows; they must not be interpreted as counts of distinct events.

| Window | Group | Reason | Distinct events |
|---|---|---|---:|
| in_sample | ceo_appointment | classification_nonroutine | 1 |
| in_sample | ceo_appointment | classification_unknown | 4 |
| in_sample | ceo_appointment | no_paired_listed_strikes | 1 |
| in_sample | ceo_appointment | no_usable_ordinary_control | 6 |
| in_sample | ceo_appointment | usable | 3 |
| in_sample | ceo_resignation | no_usable_ordinary_control | 3 |
| in_sample | cfo_appointment | classification_nonroutine | 3 |
| in_sample | cfo_appointment | classification_unknown | 14 |
| in_sample | cfo_appointment | no_usable_ordinary_control | 10 |
| in_sample | cfo_appointment | no_usable_ordinary_control_or_entry_call | 3 |
| in_sample | cfo_appointment | usable | 3 |
| in_sample | cfo_resignation | no_usable_ordinary_control | 6 |
| out_of_sample | ceo_appointment | classification_unknown | 7 |
| out_of_sample | ceo_resignation | invalid_role_former_ceo_board_resignation | 1 |
| out_of_sample | ceo_resignation | no_entry_day_call_trade | 1 |
| out_of_sample | ceo_resignation | no_usable_ordinary_control | 2 |
| out_of_sample | cfo_appointment | classification_nonroutine | 5 |
| out_of_sample | cfo_appointment | classification_unknown | 10 |
| out_of_sample | cfo_appointment | no_usable_ordinary_control | 4 |
| out_of_sample | cfo_appointment | usable | 1 |
| out_of_sample | cfo_resignation | no_usable_ordinary_control | 4 |

## Cost and ordinary-day sensitivity

The secondary ordinary definition excludes any disclosure within one trading session and leadership events within 30 calendar days. It was frozen before this run to diagnose the prior all-disclosure 30-day coverage problem. Its results cannot substitute for the primary comparison. JEV-low is a broader proxy, not established routine status.

| Window | Group | Ordinary rule | Premium haircut per side | Events | Difference |
|---|---|---|---:|---:|---:|
| in_sample | ceo_appointment | entry_clean_leadership_30d | 0.0% | 5 | -1.021% |
| in_sample | ceo_appointment | entry_clean_leadership_30d | 2.5% | 5 | -1.030% |
| in_sample | ceo_appointment | entry_clean_leadership_30d | 5.0% | 5 | -1.038% |
| in_sample | ceo_appointment | entry_clean_leadership_30d | 10.0% | 5 | -1.055% |
| in_sample | ceo_appointment | all_disclosures_30d | 0.0% | 3 | -1.814% |
| in_sample | ceo_appointment | all_disclosures_30d | 2.5% | 3 | -1.868% |
| in_sample | ceo_appointment | all_disclosures_30d | 5.0% | 3 | -1.922% |
| in_sample | ceo_appointment | all_disclosures_30d | 10.0% | 3 | -2.029% |
| in_sample | cfo_appointment | entry_clean_leadership_30d | 0.0% | 9 | -2.074% |
| in_sample | cfo_appointment | entry_clean_leadership_30d | 2.5% | 9 | -2.127% |
| in_sample | cfo_appointment | entry_clean_leadership_30d | 5.0% | 9 | -2.180% |
| in_sample | cfo_appointment | entry_clean_leadership_30d | 10.0% | 9 | -2.285% |
| in_sample | cfo_appointment | all_disclosures_30d | 0.0% | 3 | -1.114% |
| in_sample | cfo_appointment | all_disclosures_30d | 2.5% | 3 | -1.174% |
| in_sample | cfo_appointment | all_disclosures_30d | 5.0% | 3 | -1.234% |
| in_sample | cfo_appointment | all_disclosures_30d | 10.0% | 3 | -1.354% |
| out_of_sample | cfo_appointment | entry_clean_leadership_30d | 0.0% | 4 | -9.052% |
| out_of_sample | cfo_appointment | entry_clean_leadership_30d | 2.5% | 4 | -9.235% |
| out_of_sample | cfo_appointment | entry_clean_leadership_30d | 5.0% | 4 | -9.417% |
| out_of_sample | cfo_appointment | entry_clean_leadership_30d | 10.0% | 4 | -9.783% |
| out_of_sample | cfo_appointment | all_disclosures_30d | 0.0% | 1 | +0.636% |
| out_of_sample | cfo_appointment | all_disclosures_30d | 2.5% | 1 | +0.662% |
| out_of_sample | cfo_appointment | all_disclosures_30d | 5.0% | 1 | +0.689% |
| out_of_sample | cfo_appointment | all_disclosures_30d | 10.0% | 1 | +0.741% |

All other predeclared strike, expiry, entry-delay, staleness, category and horizon sensitivities, ±5% upside/downside tail frequencies, absolute moves, observed path extremes, stock-return quantiles, earnings proximity, premiums and liquidity are in `all_horizons_sensitivity.csv`. Samples differ when marks or controls are missing; a sign change across samples is not necessarily a parameter effect.

## Rubric assessment and submission viability

* **Hypothesis and novelty (30):** economically coherent, with an explicit option overlay benchmark. Selling calls on calm transitions is a plausible but familiar premium-selling idea; the distinct contribution would need convincing category-specific incremental value.
* **Analytical rigor (30):** runnable frozen protocol, separate roles, actual stock data, all horizons, paired ordinary controls, net costs, tails, dependence safeguards, exclusions and parameter sensitivity are delivered. Sparse usable events and unavailable primary intervals limit the strength of the empirical finding. Insignificance does not prove no effect.
* **Sealed replication (20):** no untouched judge window was run. Historical dates overlap previous local and teammate research, including earlier broad leadership covered-call comparisons. Those periods cannot honestly be described as pristine OOS. Frozen prediction: fragile/inconclusive under scarce controls, costs and asymmetric upside tails. A genuine judge rerun is supported by `run_study(start,end)`; no replication points are presumed.
* **Trade realism (10):** next-session-close entry respects public disclosure, observed stock replaces parity, standard calls require entry-day trades, splits are excluded, and volume capacity is reported. Daily last prints and assumed spreads remain theoretical; early assignment, dividend-driven exercise, borrow/financing and actual execution are not modeled. The stock price returns shown exclude dividends.
* **Communication (10):** clear supported/contradicted/inconclusive outcome, exact premium versus buyback accounting, explicit failure modes and a runnable saved notebook. This is an assessment of the evidence, not an estimate of the judges' numerical score.

**Recommendation:** retain this as a documented research result. Do not present it as a validated covered-call signal unless the frozen primary comparison has adequate independent events and confirms in genuinely untouched dates. A well-explained coverage/fragility result can be submitted honestly, but has a weaker path to strong analytical and replication marks than a hypothesis with sufficient observations. Neither pooling roles nor selecting a successful sensitivity is a valid rescue.

## Reproduce

Run all cells of `gator-quant-hacks-8k-options-covered-call-hypothesis.ipynb`, or `python covered_call_hypothesis.py`, from this repository with the Massive key in `.env`. Run `python test_covered_call_hypothesis.py` for offline accounting, time-value, split, staleness and volume-capacity checks. A local `.hypothesis_deps` folder supplies requests to Codex bundled Python; ordinary environments can install the project requirements.